Jenkinsfile

```diff
@@ -91,7 +99,13 @@ node {
 
 // Return the run id
 def getRunId() {
-    return powershell(returnStdout: true, script: 'Write-Output (Get-Content .\\run.log | select -First 1 | Select-String -Pattern \'.*\\"RunId\\": \\"([^\\"]*)\\"\').Matches.Groups[1].Value')
+    if (fileExists("multidataset_run.log")) {
+        return powershell(returnStdout: true, script: 'Write-Output (Get-Content .\\multidataset_run.log | select -First 1 | Select-String -Pattern \'.*\\"RunId\\": \\"([^\\"]*)\\"\').Matches.Groups[1].Value')
```

Comment:
Is this the batch execution id or run id of the first dataset executed?

Reply:
I am not sure about what we call it.
so, when the multidataset execution starts the multidataset_run.log is created and has the runId inside it. than it generates subsequent run.log for each execution.
so, if there are 3 total dataset including default. it generates multidataset_run.log and 3 run.log which gets overwritten one after the other.